app/controllers/friendships_controller.rb

```diff
@@ -14,19 +14,22 @@ def create
   end
 
   def update
-    @friendship = current_user.inverse_friendships.find_by(user_id: params[:id])
+    @friendship = current_user.friendship_requests.find_by(user_id: params[:id])
     if @friendship
       @friendship.update(confirmed: true)
+      Friendship.create(user_id: params[:user_id], friend_id: params[:id], confirmed: true)
       redirect_back(fallback_location: root_path, notice: 'Friend request accepted')
     else
       redirect_back(fallback_location: root_path, alert: 'Unable to send friend request')
     end
   end
```

Comment:
![image](https://user-images.githubusercontent.com/60448833/97220660-db095d00-17dc-11eb-8d7a-21bb243f8512.png)

- [ ]  [OPTIONAL] Follow the MVC best practices.

Green ones are OK to be in the controllers. 
For the red ones, you can create a class method in the Friendship or in the User, to put that logic. And then you can call that method itself right after the `if`.

That way, you will be following the best practices of the MVC architecture.

Reply:
This is about keeping the controllers thin and models thick.

Reply:
Thank you very much for your review